test/unit/firebase-namespace.spec.ts

```diff
@@ -349,6 +349,13 @@ describe('FirebaseNamespace', () => {
     it('should return a reference to Auth type', () => {
       expect(firebaseNamespace.auth.Auth).to.be.deep.equal(AuthImpl);
     });
+
+    it('should return a cached version of Auth on subsequent calls', () => {
+      firebaseNamespace.initializeApp(mocks.appOptions);
+      const serviceNamespace1: Auth = firebaseNamespace.auth();
+      const serviceNamespace2: Auth = firebaseNamespace.auth();
+      expect(serviceNamespace1).to.deep.equal(serviceNamespace2);
```

Comment:
all the `to.deep.equal` should be replaced with `to.equal`

Reply:
Thanks. Done.